Mathlib/Algebra/Category/Grp/Biproducts.lean

```diff
@@ -63,14 +63,10 @@ theorem binaryProductLimitCone_cone_π_app_right (G H : AddCommGrp.{u}) :
 /-- We verify that the biproduct in `AddCommGrp` is isomorphic to
 the cartesian product of the underlying types:
 -/
-@[simps! hom_apply]
 noncomputable def biprodIsoProd (G H : AddCommGrp.{u}) :
     (G ⊞ H : AddCommGrp) ≅ AddCommGrp.of (G × H) :=
   IsLimit.conePointUniqueUpToIso (BinaryBiproduct.isLimit G H) (binaryProductLimitCone G H).isLimit
 
```

Comment:
It looks like we're now missing any lemmas describing the forward direction of the isomorphism `biprodIsoProd`. Can these be restored? 

Obviously we'd prefer these lemmas to be auto-generated in some way, and if `simps` can't do this correctly then let's also have an issue tracking that.

Reply:
They are not used in Mathlib, so I propose that anyone wanting them adds them when/if they are needed.